### ISSUE TITLE
CLI `csv2json -h` results in an error

### ISSUE THREAD

**Issue by porg** [mrodrig/json-2-csv-cli#17]
## Reproduction

- In the CLI version
  - `csv2json --help`, `json2csv --help` and `json2csv --h` all work fine
  - but `csv2json -h` results in an error.

## Expected Behavior

- `csv2json -h` Shows the help screen

## Actual Behavior

- Results in an error

```bash
$ json2csv --help

# Shows the help screen fine

$ json2csv -h    

# Shows the help screen fine

$ csv2json --help

# Shows the help screen fine

$ csv2json -h    
/usr/local/lib/node_modules/@mrodrig/json-2-csv-cli/node_modules/commander/lib/command.js:581
      throw err;
      ^

TypeError: Cannot read properties of undefined (reading 'split')
    at Option.constructKeysList [as parseArg] (/usr/local/lib/node_modules/@mrodrig/json-2-csv-cli/bin/utils/utils.js:88:17)
    at Command._callParseArg (/usr/local/lib/node_modules/@mrodrig/json-2-csv-cli/node_modules/commander/lib/command.js:575:21)
    at handleOptionValue (/usr/local/lib/node_modules/@mrodrig/json-2-csv-cli/node_modules/commander/lib/command.js:674:20)
    at Command.<anonymous> (/usr/local/lib/node_modules/@mrodrig/json-2-csv-cli/node_modules/commander/lib/command.js:694:7)
    at Command.emit (node:events:520:28)
    at Command.parseOptions (/usr/local/lib/node_modules/@mrodrig/json-2-csv-cli/node_modules/commander/lib/command.js:1689:18)
    at Command._parseCommand (/usr/local/lib/node_modules/@mrodrig/json-2-csv-cli/node_modules/commander/lib/command.js:1449:25)
    at Command.parse (/usr/local/lib/node_modules/@mrodrig/json-2-csv-cli/node_modules/commander/lib/command.js:1064:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/@mrodrig/json-2-csv-cli/bin/csv2json.js:23:6)
    at Module._compile (node:internal/modules/cjs/loader:1504:14)

Node.js v22.5.1

$

```

## Background Information
- Module Version: `5.5.5`
  - Installed a global app via `npm install -g @mrodrig/json-2-csv-cli`
- Node: `v22.5.1`
- OS: macOS 13.6.9 Ventura
  - Terminal.app running zsh, nevertheless I used the `$` symbol from bash, as it's more conventional.

The issue I'm reporting is with:
- [ ] json2csv
- [x] csv2json

I have...
- [x] searched to see if an issue has already been reported.
- [x] tried upgrading to the latest version of json-2-csv (since the issue may already be fixed).
- verified that my JSON/CSV data is valid (using something like http://jsonlint.com or https://csvlint.io/).
  - does not apply as my bug occurs with `csv2json -h` which uses no input/output at all.

**Comment by porg** [comment]
@mrodrig it's a minor bug but still I'd appreciate a reaction. Thanks!

**Comment by mrodrig** [comment]
Sorry for the delay getting back to you @porg. This is definitely an issue, so I'll see if I can track it down and will plan to publish a fix for this in `5.5.6` when I pull in another fix from `json-2-csv`@`5.5.6`. Thank you for reporting it and your patience.

**Comment by mrodrig** [comment]
This should be resolved in version `5.5.6` that's on NPM now. Please let me know if you're still experiencing the error after updating.
